src/components/Tools/AdvisorCard.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="advisors">
-    <div class="advisors-content">
+    <div class="advisors-content" v-if="advisors.length === 0">
+      <span class="advisors-empty"> Sorry, you don't have an advisor yet. </span>
```

Comment:
I think you should confirm this copy @noschiff, imo the wording is pretty dev-centric and might sound strange to the user if they do have an advisor but we don't have the data.

Reply:
I took that from the design

Reply:
![image](https://user-images.githubusercontent.com/47431797/198386208-f5cc2240-5478-446e-985e-556c7078e7a1.png)
this was the design, but I'm happy to change it